File: tests/testSrc/com/intellij/lang/jsgraphql/frameworks/relay/GraphQLRelayValidationTest.java
package com.intellij.lang.jsgraphql.frameworks.relay;

import com.intellij.lang.jsgraphql.GraphQLTestCaseBase;
import com.intellij.lang.jsgraphql.GraphQLTestUtils;
import com.intellij.lang.jsgraphql.schema.library.GraphQLBundledLibraryTypes;
import org.jetbrains.annotations.NotNull;

public class GraphQLRelayValidationTest extends GraphQLTestCaseBase {
  @Override
  protected @NotNull String getBasePath() {
    return "/frameworks/relay/validation";
  }

  public void testSuppressedInspections() {
    GraphQLTestUtils.withLibrary(getProject(), GraphQLBundledLibraryTypes.RELAY, this::doHighlightingTest, getTestRootDisposable());
  }
}

File: tests/testSrc/com/intellij/lang/jsgraphql/frameworks/relay/GraphQLRelayValidationTest.kt
package com.intellij.lang.jsgraphql.frameworks.relay

import com.intellij.lang.jsgraphql.GraphQLTestCaseBase
import com.intellij.lang.jsgraphql.schema.library.GraphQLBundledLibraryTypes
import com.intellij.lang.jsgraphql.withLibrary
import com.intellij.openapi.progress.runBlockingCancellable

class GraphQLRelayValidationTest : GraphQLTestCaseBase() {
  override fun getBasePath(): String {
    return "/frameworks/relay/validation"
  }

  fun testSuppressedInspections() = runBlockingCancellable {
    withLibrary(project, GraphQLBundledLibraryTypes.RELAY) { doHighlightingTest() }
  }
}
